app/code/community/Yotpo/Yotpo/Helper/Data.php

```diff
@@ -56,7 +56,7 @@ private function renderYotpoProductBlock($thisObj, $blockName, $product = null,
         }
         if ($block != null) {
             if ($print == true) {
-                $block->toHtml();
```

Comment:
can remove if and logic be the same

Reply:
yes we can remove this.

Reply:
@ijazalise could you please remove this redundant code?